userportaldatamodel/schemas/search_schema.py

```diff
@@ -1,8 +1,17 @@
+from enum import Enum
+
 from marshmallow_sqlalchemy import SQLAlchemyAutoSchema
-from userportaldatamodel.models import Search
+from marshmallow_enum import EnumField
+from userportaldatamodel.models import FilterSourceType, Search
+
+
+# class FilterSourceTypeSchema(SQLAlchemyAutoSchema):
```

Comment:
Just curious, but why is this commented out bit is left around?

Reply:
I was attempting to include serialization and ran into obstacles.  I left it in place to show what remains to be done so someone else can pickup where I left off, but I can just as easily remove it.

Reply:
If that's the case, can you please describe exactly what is to be done in comment? I'm not as familiar with SQLAlchemy works as you do, and your description will be a great help to whoever will come to pick it up. I'm afraid that commented out lines of code just looks like something to be removed without such description.

Reply:
Additional details added to the comment and I removed necessary lines.  

Please note, this PR needs to be merged & a new release generated before amanuensis PR can be merged in (I need to the update the amanuensis poetry files with the new version of userPortalDataModel)

Reply:
Thank you for adding the comment, and my apologies for dragging this review too long. Please feel free to move forward.